clang/test/CXX/drs/cwg28xx.cpp

```diff
@@ -150,7 +150,7 @@ void g() {
 #endif
 } // namespace cwg2877
 
-namespace cwg2881 { // cwg2881: 19 tentatively ready 2024-04-19
+namespace cwg2881 { // cwg2881: 19 ready 2024-04-19
```

Comment:
Since this was added, wording has changed slightly, and a new ill-formed example was added. See https://github.com/cplusplus/CWG/blame/gh-pages/issues/2881.html
CC @Sirraide

Reply:
We’re already rejecting the ill-formed example in question, but we should probably add that as a test case.

Reply:
We do still need a test for template deduction, though, because from what I can tell, the wording is still the same, except that this is now also specified as causing template argument deduction to fail.